what4/src/What4/BaseTypes.hs

```diff
@@ -30,6 +30,9 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE  OverloadedStrings #-}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
The convention in this module is to alphabetize the `LANGUAGE` pragmas, so please alphabetize these as well.